packages/siwx/src/configs/CloudAuthSIWX.ts

```diff
@@ -18,11 +18,13 @@ import { InformalMessenger } from '../index.js'
  * WARNING: The Claud Auth is only available in EVM networks.
  */
 export class CloudAuthSIWX implements SIWXConfig {
-  private readonly localStorageKey: string
+  private readonly localAuthStorageKey: string
+  private readonly localNonceStorageKey: string
   private readonly messenger: SIWXMessenger
 
   constructor(params: CloudAuthSIWX.ConstructorParams = {}) {
-    this.localStorageKey = params.localStorageKey || '@appkit/siwx-token'
+    this.localAuthStorageKey = params.localAuthStorageKey || '@appkit/siwx-auth-token'
```

Comment:
Can we use these keys from `SafeLocalStorage` util

Reply:
I think this should be independant from SafeLocalStorage. 

But would be nice to use the key from its constants